toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/GeoViewScope.kt

```diff
@@ -292,7 +309,9 @@ public sealed class GeoViewScope protected constructor(private val geoView: GeoV
     ): ScreenCoordinate? {
         val geoViewRotation = geoView.rotation()
         val locationToScreen = when (geoView) {
-            is MapView -> geoView.locationToScreen(location)
+            is MapView -> geoView.locationToScreen(location).takeIf {
+                !it.x.isNaN() && !it.y.isNaN()
+            }
             is SceneView -> {
                 val locationToScreenResult = geoView.locationToScreen(location)
                 if (locationToScreenResult?.visibility == SceneLocationVisibility.Visible) {
```

Comment:
This line could be updated to match the above: 
```suggestion
            is SceneView -> geoView.locationToScreen(location)?.takeIf { it.visibility == SceneLocationVisibility.Visible }?.screenPoint
```